malasakit-django/pcari/signals.py

```diff
@@ -7,12 +7,64 @@
 
 from __future__ import unicode_literals
 
+from math import sqrt
+
 from django.db.models.signals import pre_delete, post_delete
+from django.db.backends.signals import connection_created
 from django.dispatch import receiver
 
 from pcari.models import History
 
 
+def make_stddev_aggregate(sample=False):
+    """
+    Create a standard deviation aggregator to pass to PySQLite. In production,
```

Comment:
Has std deviation been tested on MySQL?

Reply:
Yes--the Travis build uses MySQL to closely simulate our production environment.